Bottom: b62859314feb264e52a429fb9df630275946b02e
Top:    4c6fc33ef15e5fcbb6f4648d7613f6f22e51bb44
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:48:57 +0000

Comment object3d_draw_debug() call during expose


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 6e8c5d3..9f1d695 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2698,7 +2698,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   glDisable (GL_LIGHTING);
 
   draw_crosshair (Output.fgGC, priv);
-  object3d_draw_debug ();
+//  object3d_draw_debug ();
 
   hidgl_flush_triangles (priv->hidgl);
